lib/peg.d.ts

```diff
@@ -0,0 +1,148 @@
+// Based on PEG.js Type Definitions by: vvakame <https://github.com/vvakame>, Tobias Kahlert <https://github.com/SrTobi>, C.J. Bell <https://github.com/siegebell>
+
+declare namespace PEG {
+  function parse(input: string): any;
+
+  interface Location {
+    line: number;
+    column: number;
+    offset: number;
+  }
+
+  interface LocationRange {
+    start: Location;
+    end: Location;
+  }
+
+  class SyntaxError {
+    line: number;
+    column: number;
+    offset: number;
+    location: LocationRange;
```

Comment:
This is quite strange... `location` should be enough. Need to check if the `line/column/offset` are really filled in the error

Reply:
it might need revisiting, I've just copied the definitions from DefinitelyTyped here

Reply:
Just checked this, `line/column/offset` is definedly an error:
https://github.com/peggyjs/peggy/blob/2a37f1f02b57a3c5a2277d2735b2773de30d2aa4/lib/compiler/passes/generate-js.js#L754-L764

Reply:
@mingun can you file a new bug please?  I'll mark it release-blocking

Reply:
#29